parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -257,6 +261,29 @@ ensureRunc() {
 
 cleanUpGPUDrivers() {
     rm -Rf $GPU_DEST /opt/gpu
+    
+    # Remove cached GPU device plugin downloads on non-GPU nodes
+    if [ -d "/opt/nvidia-device-plugin/downloads" ]; then
```

Comment:
we should always remove them no ?  we don't need cached packages even when we install them

Reply:
You're right. It was removed in that case too, but the comments weren't right. Simplified that logic.